desktop/src/renderer/src/App.tsx

```diff
@@ -1,127 +1,149 @@
+		const ipStatusOK = await api.checkIpRange();
+		setShowVPNPopup(!ipStatusOK);
+	};
 
-  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
-  useEffect(() => {
-    // Check for VPN or gov Network use on load, and every 5 seconds
-    handleIPStatusUpdate();
-    const interval = setInterval(handleIPStatusUpdate, 5 * 1000);
+	// biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
+	useEffect(() => {
+		// Check for VPN or gov Network use on load, and every 5 seconds
+		handleIPStatusUpdate();
+		const interval = setInterval(handleIPStatusUpdate, 5 * 1000);
 
-    return () => {
-      clearInterval(interval);
-    };
-  }, []);
+		return () => {
+			clearInterval(interval);
+		};
+	}, []);
 
-  return (
-    <BrowserRouter>
-      <Grid container sx={{ height: "100vh" }}>
-        {showVPNPopup && <VPNPopup />}
-        <Grid size={2}>
-          <SideNav accessToken={accessToken} idToken={idToken} />
-        </Grid>
-        <Grid size={10}>
-          <Header
-            title="Digital Archives Transfer Service"
-            logoLinkElement={
-              <Button onClick={() => setLeavePageModalOpen(true)} />
-            }
-          />
-          <Routes>
-            <Route
-              path="/"
-              element={<HomePage authenticated={!!accessToken} />}
-            />
-            <Route
-              path="/file-list"
-              element={<FileListPage authenticated={!!accessToken} />}
-            />
-            <Route path="/send-records" element={<SendRecordsPage />} />
-            <Route
-              path="/send-records/lan/instructions"
-              element={<LanInstructionsPage />}
-            />
-            <Route path="/send-records/lan" element={<LanTransferPage />} />
-          </Routes>
-          <ToastContainer
-            position="bottom-left"
-            autoClose={5000}
-            hideProgressBar
-            pauseOnHover
-          />
-        </Grid>
-        <LeavePageModal
-          open={leavePageModalOpen}
-          onClose={() => setLeavePageModalOpen(false)}
-          onConfirm={onConfirmLeavePage}
-        />
-      </Grid>
-    </BrowserRouter>
-  );
+	return (
+		<BrowserRouter>
+			<Grid
+				container
+				sx={{ height: '100vh' }}
+			>
+				{showVPNPopup && <VPNPopup />}
+				<Grid size={2}>
+					<SideNav
+						accessToken={accessToken}
+						idToken={idToken}
+					/>
+				</Grid>
+				<Grid size={10}>
+					<Header
+						title='Digital Archives Transfer Service'
+						logoLinkElement={
+							<Button onClick={() => setLeavePageModalOpen(true)} />
+						}
+					/>
+					<Grid container>
+						<Grid size={2} />
+						<Grid
+							size={8}
+							sx={{ paddingTop: 3 }}
+						>
+							<Routes>
+								<Route
+									path='/'
+									element={<HomePage authenticated={!!accessToken} />}
+								/>
+								<Route
+									path='/file-list'
+									element={
+										<TokenContext.Provider value={accessToken}>
```

Comment:
If we're going to use context, can we give this context to add pages and replace the authenticated prop on each page